nilearn/masking.py

```diff
@@ -656,7 +673,73 @@ def compute_multi_gray_matter_mask(target_imgs, threshold=.5,
 
     threshold: float, optional
         The value under which the MNI template is cut off.
-        Default value is 0.5
+        Default value is 0.5.
+
+    connected: bool, optional
+        if connected is True, only the largest connect component is kept.
+        Default is True
+
+    opening: bool or int, optional
+        if opening is True, a morphological opening is performed, to keep
+        only large structures.
+        If opening is an integer `n`, it is performed via `n` erosions.
+        After estimation of the largest connected constituent, 2`n` closing
```

Comment:
What motivates this rather weird behavior ?

Reply:
This parameter has been set already for the the original function which is being deprecated, as you can see for instance from [here](https://nilearn.github.io/modules/generated/nilearn.masking.compute_multi_gray_matter_mask.html).